vizro-core/docs/pages/user-guides/figure.md

```diff
@@ -105,3 +105,238 @@ To add a `Figure` to your page, do the following:
+        from vizro.figures import kpi_card, kpi_card_reference
+
+        df = pd.DataFrame([[67434, 65553, "A"], [6434, 6553, "B"], [34, 53, "C"]], columns=["Actual", "Reference", "Category"])
+
+        page = vm.Page(
+            title="KPI Indicators",
+            layout=vm.Layout(grid=[[0, 1, 2, 3], [4, 5, 6, 7], [-1, -1, -1, -1], [-1, -1, -1, -1]]),
+            components=[
+                vm.Figure(figure=kpi_card(data_frame=df, value_column="Actual", title="KPI with value")),
+                vm.Figure(figure=kpi_card(data_frame=df, value_column="Actual", title="KPI with aggregation", agg_func="mean")),
+                vm.Figure(
+                    figure=kpi_card(
+                        data_frame=df,
+                        value_column="Actual",
+                        title="KPI with custom formatting",
+                        value_format="${value:.2f}",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card(
+                        data_frame=df,
+                        value_column="Actual",
+                        title="KPI with icon",
+                        icon="shopping_cart",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card_reference(
+                        data_frame=df,
+                        value_column="Reference",
+                        reference_column="Actual",
+                        title="KPI with reference (neg)",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card_reference(
+                        data_frame=df,
+                        value_column="Actual",
+                        reference_column="Reference",
+                        title="KPI with reference (pos)",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card_reference(
+                        data_frame=df,
+                        value_column="Reference",
+                        reference_column="Actual",
+                        title="KPI with reference and custom formatting",
+                        value_format="{value:.2f}$",
+                        reference_format="{delta:.2f}$ vs. last year ({reference:.2f}$)",
+                    )
+                ),
+                vm.Figure(
+                    figure=kpi_card_reference(
+                        data_frame=df,
+                        value_column="Actual",
+                        reference_column="Reference",
+                        value_format="${value:.2f}",
+                        title="KPI with reference and icon",
+                        icon="shopping_cart",
+                    ),
+                ),
+            ],
+            controls=[vm.Filter(column="Category")],
+        )
+
+
+        dashboard = vm.Dashboard(pages=[page])
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
```

Comment:
This is not in sync yet. Want to create the yaml example if we agree on the pydantic example 👍 My idea was to show on one page the different variations of KPI Card configurations. In the end, the users can decide to which one to use and mix them, but I think it's important to have one visual overview

Reply:
![kpi_cards](https://github.com/mckinsey/vizro/assets/90609403/93ffe980-bccb-4dfb-9520-0fa1fbaea394)

Reply:
This is a wonderful set of examples I think! Goes through a very representative set of options clearly 👍